### PR TITLE
docs: update jina embeddings integration

### PR DESCRIPTION
**Pull request by bwanglzu**
Update documentation of jina embeddings:

1. Created a detailed model list with tech summary.
2. add description of task adapters of `jina-embeddings-v3` and how to use them.
3. add short description of MRL performance to give user a hint how to balance between storage and performance.
4. updated code snnipt.

### CODE REVIEW

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (CatStark) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:22]:
```suggestion
|  jina-embeddings-v2-base-es |  768 |  Spanish & English |  No  |  8192 | 
```

**Review by CatStark — CHANGES_REQUESTED** [review]

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (Anush008) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:11]:
```suggestion
Qdrant is compatible with [Jina AI](https://jina.ai/) embeddings. You need an API key from [Jina AI](https://jina.ai/embeddings/) to get embeddings.
```

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (Anush008) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:25]:
```suggestion
> Jina recommend using `jina-embeddings-v3` as it is the latest and most performant embedding model released by Jina AI.
```

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (Anush008) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:58]: Probably mention the free trial in the texts above instead of this comment.

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (Anush008) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:25]:
```suggestion
> Jina recommends using `jina-embeddings-v3` as it is the latest and most performant embedding model released by Jina AI.
```

**Comment by bwanglzu** [comment]
@Anush008 thanks for your review, please do not merge right now we might make a small change to api schema

**Comment by bwanglzu** [comment]
@Anush008 PR updated, two small changes since last review:

1. rename `task_type` to `task` to adopt API change.
2. remove "free api key" from the code sample

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (Anush008) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:46]:
```suggestion
## Example
```

File: qdrant-landing/content/documentation/embeddings/jina-embeddings.md
Comment (Anush008) [inline qdrant-landing/content/documentation/embeddings/jina-embeddings.md:13]:
```suggestion
Qdrant users can receive a 10% discount on Jina AI APIs by using the code **QDRANT**.
```

**Review by Anush008 — APPROVED** [review]
Thank you.

**Pull request merged**
